test/com/dotmarketing/util/IntegrationTestInitService.java

```diff
@@ -0,0 +1,37 @@
+package com.dotmarketing.util;
+
+import com.dotmarketing.servlets.test.ServletTestRunner;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * Sets up the web environment needed to execute integration tests without a server application
+ * Created by nollymar on 9/29/16.
+ */
+public class IntegrationTestInitService {
+    private static IntegrationTestInitService service = null;
+
+    private static AtomicBoolean initCompleted;
+
+    private IntegrationTestInitService() {
+        initCompleted = new AtomicBoolean(false);
+    }
+
+    public static IntegrationTestInitService getInstance() {
+        if (service == null) {
+            service = new IntegrationTestInitService();
+        }
```

Comment:
Really it is not a thread safe way to implement a singleton.
here explain my point: 
http://crunchify.com/thread-safe-and-a-fast-singleton-implementation-in-java/